package.json

```diff
@@ -1,22 +1,26 @@
 {
   "name": "@nimiq/nano-api",
-  "version": "0.0.9",
+  "version": "0.1.0-beta.1",
   "description": "Nano Network API",
-  "main": "dist/nano-network-api.umd.js",
-  "module": "dist/nano-network-api.es.js",
+  "main": "dist/NanoApi.umd.js",
+  "module": "dist/NanoApi.es.js",
+  "types": "types/index.d.ts",
   "repository": "git@github.com:nimiq/nano-api.git",
   "author": "Nimiq",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "rollup -c"
+    "build": "rm -rf build && rm -rf dist && tsc && rollup -c"
   },
   "files": [
-    "dist"
+    "dist",
+    "types/index.d.ts"
   ],
   "dependencies": {
-    "@nimiq/utils": "^0.0.2"
+    "@nimiq/utils": "^0.2.0"
   },
   "devDependencies": {
-    "rollup": "^0.65.0"
+    "@nimiq/core-types": "^1.4.2",
+    "rollup": "^0.65.0",
+    "typescript": "^3.3.3333"
```

Comment:
What kind of version is this?

Reply:
https://github.com/nimiq-network/core/pull/479

> I would really like to know how many developer hours have been wasted worldwide investigating about this version number... with great power comes great responsibility :)
--

Reply:
Trolling at its best :rofl: